src/app/components/welcome-page/welcome-page.component.scss

```diff
@@ -34,34 +26,26 @@
   grid-template-rows: 1fr;
   grid-template-areas: 
     "welcomeSign";
-  background-color: $darkBlue;
   color: $lightTextColor;
   text-transform: uppercase;
   text-align: center;
-  align-self: stretch;
-  padding: 6rem;
-  border-left: 1rem solid $orange;
 }
 
 .welcomeSign {
   grid-area: welcomeSign;
-  font-size: 2,5em;
-  letter-spacing: 0.5rem;
-  strong {
-    color: $orange;
-  };
-  text-shadow: 1px 1px 4px black;
+  @include title($darkBlue, #659de0, 0.2rem);
```

Comment:
This feels odd. You are pretty often using variables and some hardcoded colors. Based on some reading, designers often use color palettes with a limited amount of colors. The reasons are:
- Too many colors make it hard to maintain
- Too many colors doesn't make it possible to create variable for all
- Simple designs (which are modern nowadays) are not possible

I'm not saying this is an issue - you are the one who plays with design here. However. it may be too much at some point and it's better to keep it simple.